### DISCUSSION TITLE
Makefile Issue: VScode download for Windows

### DISCUSSION THREAD

**Discussion by Shane0xM** [Cyfrin/foundry-full-course-cu#1824] [General]
Hello, 

Working through Lesson 20 in "Foundry Fund Me" section and have run into issues installing / configuring Makefile.

I have installed via the extensions in my terminal, but when I call "make" I get "command not found" error. 

I attempted to download MinGW installation manager to download this to local path. Still not resolved. 

Would appreciate any advice here on best approach for setup on Windows.

Thanks, 
SM

**Comment by Shane0xM** [comment]
Answering my own question here for others with similar issues:

Found out that on windows 10 or 11, you can run the command `winget install ezwinports.make` in the command line or powershell to quickly install it.

Then restart your terminal and you will be able to use the command 'make'.

**Reply by EngrPips** [reply]
Awesome. I'm glad you unblocked yourself, and thanks for dropping the solution here to help others.

**Reply by helalmerchan** [reply]
> Answering my own question here for others with similar issues:
> 
> Found out that on windows 10 or 11, you can run the command `winget install ezwinports.make` in the command line or powershell to quickly install it.
> 
> Then restart your terminal and you will be able to use the command 'make'.

Thank you, the command `winget install ezwinports.make` helped me.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.

**Reply by Monster-Three** [reply]
hey,bro. I can't use 'winget install ezwinports.make' to install make,but I use 'sudo apt install make' to install make,after I put my password in,it works!(by the way,I use Gemini to help me solve this problem.)

**Reply by EngrPips** [reply]
> hey,bro. I can't use 'winget install ezwinports.make' to install make,but I use 'sudo apt install make' to install make,after I put my password in,it works!(by the way,I use Gemini to help me solve this problem.)

Awesome, keep crushing. it. Yeah Gemini is decent to help with many coding stuffs(web3)
